chain/jsonrpc/src/lib.rs

```diff
@@ -250,6 +241,9 @@ impl JsonRpcHandler {
             "EXPERIMENTAL_changes_in_block" => self.changes_in_block(request.params).await,
             "EXPERIMENTAL_check_tx" => self.check_tx(request.params).await,
             "EXPERIMENTAL_genesis_config" => self.genesis_config().await,
+            "EXPERIMENTAL_light_client_proof" => {
```

Comment:
what is this change for

Reply:
Alphabetical ordering of methods, we found it to be easier to read and maintain